pages/views/introduction/index.njk

```diff
@@ -1,6 +1,8 @@
 ---
 layout: content
 title: Installation Guide
+seoTitle: ESL Installation Guide and Configuration
+description: Get started with installing Exadel Smart Library on the project. Configuring and using ESL components.
```

Comment:
Configuring and using ESL components can be rewritten like : "Configuration and usage of ESL components"